ami/app.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask
+from flask_restful import Api
+from resources.bems import Bems
```

Comment:
leave one line between third party module and local module

Reply:
Fix in 26cea38.